client/scraper/metadata.py

```diff
@@ -69,7 +114,18 @@ def get_favicon_url(self, soup):
             icon_link = icon_field['href'].encode('utf-8')
         return icon_link
 
-    def general_parse_content(self, response):
+    def generic_fetch_site_data(self, sanitized_url, status_code):
+        response = Response()
+
+        request = requests.get(sanitized_url)
+        redirect_url = request.url
+        sanitized_url = url_utils.remove_url_fragments(redirect_url)
+
+        provider_url = url_utils.get_domain_url(sanitized_url)
+        response.set_content(request.headers, request.content, request.status_code, sanitized_url, provider_url)
+        return response
+
+    def generic_parse_content(self, response):
```

Comment:
I talked to wenhao, he said it's better to have consistency in the json response so the values which aren't available are NULL instead. Embed.ly seems to do this too